ImplementationGuide/markdown/ISiKundSMART/Schritt6RefreshToken.md

```diff
@@ -1,23 +1,19 @@
 # Schritt 6: Refresh Token & Revocation
 
----
-### Informativ
----
-
 ## Refresh Tokens
 
 Die Gültigkeitsdauer eines Access-Tokens kann beliebig durch das bestätigungsrelevante System gewählt werden. Mit Hinblick auf [RFC6819 - OAuth 2.0 Threat Model and Security Considerations - Limited Access Token Lifetime](https://datatracker.ietf.org/doc/html/rfc6819#section-3.1.2) soll die Gültigkeitsdauer auf Minuten oder Stunden beschränkt werden. Das Gültigkeitsdatum wird im Access Token durch den Parameter "expires_in" angegeben. Alternativ kann die Gültigkeit des Tokens über den Introspection-Endpunkt abgefragt werden.
```

Comment:
issue: das Gültigkeitsdatum befindet sich nach meinem Verständnis nicht in dem Access-Token, da dieser opaque ist. Das Gültigkeitsdatum wird dem Client "nur" in der Response beim Abruf des Tokens als ein weiterer Parameter mitgegeben https://hl7.org/fhir/smart-app-launch/STU2.1/app-launch.html#response-5